### DISCUSSION TITLE
Codepage

### DISCUSSION THREAD

**Discussion by Stepan-H** [WinMerge/winmerge#806] [Q&A] [answered]
I have text files with codepade Windows-1250, but WinMerge opens files as Windows-1252. If I set in settings codepage to Windows-1250, WinMerge still opens files as Windows-1252.

Am I doing something wrong or is there a bug in WinMerge?

**Comment by sdottaka** [comment, marked as answer]
What is set in the Codepage category of the Options dialog?

![image](https://user-images.githubusercontent.com/98126/119833830-6517d080-bf3a-11eb-8877-f6b453cc3ef3.png)

**Comment by Stepan-H** [comment]
Ah, thank you. I had set "Detect codepage for text files with mlang.dll...". After I switch it off, files opens with 1250...

I'm used to applications recognizing the correct character set, which WinMerge obviously can't... 
Fortunately, I'm comparing files that have same character set. I can't imagine how it would (not) work if I had a project with many files of different character sets.
